src/domStuff.js

```diff
@@ -0,0 +1,70 @@
+
+import basic from './fetchData';
+
+
+function clearDisplay() {
+  const city = document.querySelector('#city');
+  city.innerHTML = '';
+  const description = document.querySelector('#desc');
+  description.innerHTML = '';
+  const temp = document.querySelector('#temp');
+  temp.innerHTML = '';
+
+  const icon = document.querySelector('#icon');
+  icon.src = '';
+  const toggle = document.querySelector('.toggle ');
+  toggle.removeAttribute('id');
+}
+
+function displayError(err) {
+  clearDisplay();
+  const city = document.querySelector('#city');
+  city.innerHTML = `${err}`;
+}
+
+
+function display(data) {
+  const city = document.querySelector('#city');
+  city.innerHTML = data.name;
+  const description = document.querySelector('#desc');
+  description.innerHTML = data.weather[0].description;
+  const temp = document.querySelector('#temp');
+  temp.innerHTML = `${Math.round(data.main.temp)} C`;
+
+  const icon = document.querySelector('#icon');
+  icon.src = `http://openweathermap.org/img/wn/${data.weather[0].icon}@2x.png`;
+
+  const toggle = document.querySelector('.toggle ');
+  toggle.id = 'active';
+}
+
+
+function addEventListener() {
+  const submitButton = document.querySelector('#checkWeather');
+  submitButton.addEventListener('click', () => {
+    const city = document.querySelector('#userInput').value;
+    basic(city)
+      .then(data => display(data))
+      .catch(err => displayError(err));
+  });
+
+  const celcius = document.querySelector('#c');
```

Comment:
Let us separate the processing of temperature data into another non-DOM related module. Probably we can have a `utils.js` or a `helpers.js` file that will contain all the necessary mathematical operation we need for our incoming data. For instance (please see my suggested UI improvement in the main comment) we can also have a function in that file that processes dates so we can show dates in a more readable format. This is on top of the function that converts between celsius and fahrenheit.